api/app/registration/api.py

```diff
@@ -1,21 +1,175 @@
 from datetime import datetime
-import traceback
-from flask_restful import fields, marshal_with
 import flask_restful as restful
 from sqlalchemy.exc import IntegrityError
-from sqlalchemy.exc import SQLAlchemyError
-from app.utils import errors
-from app import LOGGER
 from app.registration.models import RegistrationSection
 from app.registration.models import RegistrationQuestion
 from app.registration.models import RegistrationForm
-from app.registration.models import Offer
 from app.registration.mixins import RegistrationFormMixin, RegistrationSectionMixin, RegistrationQuestionMixin
+from app.utils.auth import verify_token
+import traceback
+from flask import g, request
+from flask_restful import  fields, marshal_with
+from sqlalchemy.exc import SQLAlchemyError
 from app.events.models import Event
+from app.registration.models import Offer
+from app.registration.mixins import RegistrationMixin
 from app.users.models import AppUser
-from app import db
-from app.utils.auth import auth_required, admin_required, verify_token
-from flask import g, request
+from app import db, LOGGER
+from app.utils import errors
+from app.utils.auth import auth_required, admin_required
+
+OFFER_EMAIL_BODY = """
+Dear {} {} {},
+
+Congratulations on you offer
+
+Offer Details \n
+user_id {} \n
+event_id {} \n
+offer_date {} \n
+expiry_date {} \n
+payment_required {} \n
+travel_award {} \n
+accommodation_award {} \n
+
+Kind Regards,
+The Baobab Team
+"""
+
+def offer_info(offerEntity):
```

Comment:
Style: Please use underscores in variable names - so this should be `offer_entity`